<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Avogadro::QtPlugins::SymmetryWidget</class>
 <widget class="QWidget" name="Avogadro::QtPlugins::SymmetryWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Symmetry</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>391</width>
     <height>461</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="esTab">
    <attribute name="title">
     <string>Molecule</string>
    </attribute>
    <widget class="QLabel" name="esLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>341</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Symmetrically equivalent atoms:</string>
     </property>
    </widget>
    <widget class="QTableView" name="esTable">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>341</width>
       <height>341</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderCascadingSectionResizes">
      <bool>true</bool>
     </attribute>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>390</y>
       <width>341</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayoutMolecule">
      <item>
       <widget class="QCheckBox" name="lockSymmetryCheckBox">
        <property name="text">
         <string>Lock Symmetry</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacerMolecule">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="symmetrizeMoleculeButton">
        <property name="text">
         <string>Symmetrize</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="operationsTab">
    <attribute name="title">
     <string>Operations</string>
    </attribute>
    <widget class="QLabel" name="operationsLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>341</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Symmetry elements:</string>
     </property>
    </widget>
    <widget class="QTableView" name="operationsTable">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>342</width>
       <height>341</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderCascadingSectionResizes">
      <bool>true</bool>
     </attribute>
    </widget>
   </widget>
   <widget class="QWidget" name="subgroupsTab">
    <attribute name="title">
     <string>Subgroups</string>
    </attribute>
    <widget class="QLabel" name="subgroupsLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>341</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Subgroups:</string>
     </property>
    </widget>
    <widget class="QTreeView" name="subgroupsTree">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>341</width>
       <height>341</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>390</y>
       <width>341</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayoutSubgroups">
      <item>
       <spacer name="horizontalSpacerSubgroups">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="selectSubgroupButton">
        <property name="text">
         <string>Select</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="pointGroupLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>C&lt;sub&gt;1</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>539</y>
     <width>391</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="toleranceLabel">
      <property name="text">
       <string>Tolerance:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="toleranceCombo">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <item>
       <property name="text">
        <string>Tight</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Normal</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Loose</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="detectSymmetryButton">
      <property name="text">
       <string>Detect Symmetry</string>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
